src/Calculator/Views/UnitConverter.xaml

```diff
@@ -615,7 +622,8 @@
                                          x:Uid="RefreshButtonText"
                                          Foreground="{ThemeResource SystemControlHyperlinkBaseHighBrush}"
                                          Click="CurrencyRefreshButton_Click"/>
-                        <TextBlock Margin="0,7,0,0" Style="{ThemeResource CaptionTextBlockStyle}">
+                        <TextBlock x:Name="CurrencySecondaryStatusTextBlock" Margin="0,7,0,0"
```

Comment:
Install the XAML Styler extension so you get automatic formatting of XAML files.  See [README.md](/README.md).

Reply:
Super useful.  Thanks